pkg/aws/client.go

```diff
@@ -335,14 +335,21 @@ func (c *awsClient) GetAccessKeyFromStack(stackName string) (*AccessKey, error)
 // ValidateQuota
 func (c *awsClient) ValidateQuota() (bool, error) {
 	for _, quota := range serviceQuotaServices {
-		ok, err := CheckQuota(c, quota)
+		serviceQuotas, err := ListServiceQuotas(c, quota.ServiceCode)
 		if err != nil {
-			return false, fmt.Errorf("Error validating AWS quota: %s %v", quota.ServiceCode, err)
+			return false, fmt.Errorf("error listing AWS service quotas: %s %v", quota.ServiceCode, err)
 		}
-		if !ok {
-			return false, fmt.Errorf("Service %s quota code %s %s not valid",
-				quota.ServiceCode, quota.QuotaCode, quota.QuotaName)
+
+		serviceQuota, err := GetServiceQuota(serviceQuotas, quota.QuotaCode)
+		if err != nil {
+			return false, fmt.Errorf("error getting AWS service quota: %s %v", quota.ServiceCode, err)
 		}
+
+		if *serviceQuota.Value < 100 {
+			return false, fmt.Errorf("service %s quota code %s %s not valid, expected quota %f, but got %f",
```

Comment:
why not whole integers

Reply:
Thanks for the reivew. I am not very familiar with service quota values, are they all integers? But the value type is float64, I am not sure if it is safe enough to cast to int.